powersimdata/input/export_data.py

```diff
@@ -119,9 +125,12 @@ def export_case_mat(grid, filepath, storage_filepath=None):
             }
         }
 
-        savemat(storage_filepath, mpc_storage, appendmat=False)
+    if filepath:
```

Comment:
In general I recommend `if x is not None:` instead of `if x:`. The latter works fine in this case, but I think is less idiomatic. On a more practical level, `if x` depends on the type of `x` (including whether it has a `__bool__` method), so adds extra complexity.

Reply:
Sounds good! Thanks!